test/documentation/README.md

```diff
@@ -0,0 +1,117 @@
+# bio documentation
+
+Currently, we can only build the documentation on *nix systems.
+
+We offer two versions of our documentation, one intended for the user (doc_usr) and one intended for the
+library-developer and maintainer (doc_dev) of bio, which contains the documentation of internals.
+
+## How to configure:
+
+```bash
+mkdir <bio-build-dir>/documentation
+cd <bio-build-dir>/documentation
+
+cmake <bio-dir>/test/documentation
+```
+
+## How to build:
+
+Prerequisites: configuring the documentation
+
+```bash
+cd <bio-build-dir>/documentation
+
+# build user and developer documentation
+cmake --build .
+
+# or build user documentation only
+cmake --build . --target doc_usr
+
+# or build developer documentation only
+cmake --build . --target doc_dev
+```
+
```

Comment:
Is anything below this point relevant to our users?

Can we hide it in the regular user documentation?

Reply:
I removed the packaging and installation documentation. Testing is nice I though